### PR TITLE
sync avalanchego flags

### PR DESCRIPTION
**Pull request by ceyonur**
Syncs the page with latest avalanchego flags.

### CODE REVIEW

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 1]
```diff
@@ -898,86 +947,90 @@ default should only be done on private networks or local network. Defaults to
 Fraction of time a validator must be online to receive rewards. Defaults to
 `0.8`. This can only be changed on a local network.
 
-### Snow Parameters
+#### `--uptime-metric-freq` (duration)
 
-#### `--snow-avalanche-batch-size` (int)
+Frequency of renewing this node's average uptime metric. Defaults to `30s`.
+
+#### Snow Parameters
```
[the remark this answers is not in the document]
Comment (ceyonur) [reply docs/nodes/maintain/avalanchego-config-flags.md:954]: yep moved under to Consensus Paramaters

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 2]
```diff
@@ -1104,6 +1157,10 @@ Must be in \[0,1\]. Defaults to `0.9`.
 Node will report unhealthy if more than this portion of message sends fail. Must
 be in \[0,1\]. Defaults to `0.25`.
 
+#### network-health-max-outstanding-request-duration` (duration)
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:1160]: -- for consistency?

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 3]
```diff
@@ -1128,34 +1193,47 @@ AvalancheGo.
...
-Half life to use for the CPU tracker. Larger half life --> CPU usage metrics
-change more slowly. Defaults to `15s`.
-
 ##### `throttler-inbound-cpu-validator-alloc` (float)
 
 Number of CPU allocated for use by validators. Value should be in range (0, total core count].
 Defaults to half of the number of CPUs on the machine.
 
-##### `--throttler-inbound-cpu-at-large-alloc` (float)
+##### `throttler-inbound-cpu-max-recheck-delay` (duration)
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:1201]: -- for consistency?

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 4]
```diff
@@ -898,86 +947,90 @@ default should only be done on private networks or local network. Defaults to
...
 
-#### `snow-max-processing` (int)
+##### `snow-max-processing` (int)
 
 Maximum number of processing items to be considered healthy. Reports unhealthy
 if more than this number of items are outstanding. The value must be at least
 `1`. Defaults to `1024`.
 
-#### `snow-max-time-processing` (duration)
+##### `snow-max-time-processing` (duration)
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:1021]: -- for consistency?

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 5]
```diff
@@ -898,86 +947,90 @@ default should only be done on private networks or local network. Defaults to
...
 
 Maximum amount of time an item should be processing and still be healthy.
 Reports unhealthy if there is an item processing for longer than this duration.
 The value must be greater than `0`. Defaults to `2m`.
 
-#### `snow-mixed-query-num-push-vdr` (uint)
+##### `snow-mixed-query-num-push-vdr` (uint)
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:1027]: -- for consistency?

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 6]
```diff
@@ -898,86 +947,90 @@ default should only be done on private networks or local network. Defaults to
...
 
 If this node is a validator, when a container is inserted into consensus, send a
 Push Query to this many validators and a Pull Query to the others. Must be <= k.
 Defaults to `10`.
 
-#### `snow-mixed-query-num-push-non-vdr` (uint)
+##### `snow-mixed-query-num-push-non-vdr` (uint)
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:1033]: -- for consistency?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 7]
```diff
@@ -898,86 +947,90 @@ default should only be done on private networks or local network. Defaults to
...
+##### `--snow-rogue-commit-threshold` (int)
 
 Snow consensus defines `beta2` as the number of consecutive polls that a rogue
 transaction must increase its confidence for it to be accepted. This parameter
 lets us define the `beta2` value used for consensus. This should only be changed
 after careful consideration of the tradeoffs of Snow consensus. The value must
 be at least `beta1`. Defaults to `20`.
 
-#### `snow-optimal-processing` (int)
+##### `snow-optimal-processing` (int)
```
[the remark this answers is not in the document]
Comment (ceyonur) [reply docs/nodes/maintain/avalanchego-config-flags.md]: good catches, updated them

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**